src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { filterSlice } from '../../features/filter';
+import { useAppSelector } from '../../app/store';
+import { Filter } from '../../types/Filter';
```

Comment:
Ensure that the `Filter` type includes the values `ALL`, `ACTIVE`, and `COMPLETED` as used in the `<select>` options. If these values do not match, it could lead to runtime errors.